src/lib/RegistrationSchema.svelte

```diff
@@ -0,0 +1,148 @@
+<script lang="ts">
+    import {MaterialType} from "./model/registration-enums";
+    import {Body, fetch} from "@tauri-apps/api/http";
+    import {TextInputDto} from "./model/text-input-dto";
+    import {invoke} from "@tauri-apps/api/tauri";
+
+    export let workingTitle: string
+
+    const materialTypes = Object.keys(MaterialType)
+
+    let successMessage: string
+    let errorMessage: string
+
+    let materialType: string = materialTypes.at(0)
+    let fraktur: boolean = false
+    let sami: boolean = false
+    let name: string
+
+    $: {
+        const newPath = workingTitle.split('/').at(-1)
+        if (newPath) name = newPath
+        successMessage = ''
+        errorMessage = ''
```

Comment:
Vil ikke dette resette successmessage og errormessage med en gang de blir satt?

Reply:
Så vidt jeg klarte å lese meg fram til (og testet) så skjer den '$' bare "[whenever the values that they depend on have changed](https://svelte.dev/docs/svelte-components#script-3-$-marks-a-statement-as-reactive)". Tolker det slik at siden den ikke avhenger av message-stringene så skjer det ingenting når de blir endret. Eneste variabel den er avhengig av sådan er "workingTitle".